### PR TITLE
OTC-64: RfC 112 Acquring of claims

### PR DESCRIPTION
**Pull request by malinowskikam**
Changes:
-Added option to fill ClaimsActivity with JSON data (for restoring claims)
-Changed how claim restore works (to fit the rfc)
-Added functionality to get disease code with name
-Changed some layouts to fit new funcionality
-Added changes from build variants for compability

### CODE REVIEW

**Comment by malinowskikam** [comment]
@dragos-dobre Can you review?

File: claimManagement/src/main/java/org/openimis/imisclaims/SQLHandler.java [hunk 1]
```diff
@@ -9,17 +9,19 @@
...
 public class SQLHandler extends SQLiteOpenHelper{
 
 	private static final String DB_NAME = MainActivity.Path + "Mapping.db3";
-	private static final String CreateTable = "CREATE TABLE tblMapping(Code text,Name text,Type text);";
-	private static final String CreateTableControls = "CREATE TABLE tblControls(FieldName text, Adjustibility text);";
-	private static final String CreateTableClaimAdmins = "CREATE TABLE tblAdministrators(Code text, Name text);";
-	private static final String CreateTableReferences = "CREATE TABLE tblReferences(Code text, Name text, Type text, Price text);";
+	private static final String CreateTable = "CREATE TABLE IF NOT EXISTS tblMapping(Code text,Name text,Type text);";
+	private static final String CreateTableControls = "CREATE TABLE IF NOT EXISTS tblControls(FieldName text, Adjustibility text);";
+	private static final String CreateTableClaimAdmins = "CREATE TABLE IF NOT EXISTS tblAdministrators(Code text, Name text);";
```
Comment (dragos-dobre) [inline claimManagement/src/main/java/org/openimis/imisclaims/SQLHandler.java:23]: I asked to change tblAdministrators to tblClaimAdmins in another PR. Please do these changes to be more precise. Thanks.

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: claimManagement/src/main/java/org/openimis/imisclaims/SQLHandler.java (hunk 1 above)
Comment (malinowskikam) [reply claimManagement/src/main/java/org/openimis/imisclaims/SQLHandler.java]: I changed the table name to tblClaimAdmins

Comment (malinowskikam) [reply claimManagement/src/main/java/org/openimis/imisclaims/SQLHandler.java]: I changed that.

File: claimManagement/src/main/java/org/openimis/imisclaims/ClaimActivity.java [hunk 2]
```diff
@@ -535,6 +540,81 @@ private void ClearForm(){
         etClaimAdmin.requestFocus();
     }
 
+    private void fillForm(JSONObject obj)
+    {
+        try {
+            String newClaimNumber = "@" + obj.getString("claim_number");
```
Comment (dragos-dobre) [inline claimManagement/src/main/java/org/openimis/imisclaims/ClaimActivity.java:546]: The '@' prefix is configured in Web.config in WebApp. We should be able to change easily this prefix. Nepal put 're' as prefix. Please add this in strings.xml and take the value from there.

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: claimManagement/src/main/java/org/openimis/imisclaims/ClaimActivity.java (hunk 2 above)
Comment (malinowskikam) [reply claimManagement/src/main/java/org/openimis/imisclaims/ClaimActivity.java]: I replaced it with a resource, so now i can be easily changed.

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**